Sources/XcodeGenKit/PathKitExtension.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import PathKit
+
+extension Path {
+
+    /// Treat this as a resource instead of a normal directory.
+    var isNonFolderDirectory: Bool {
+
+        if isFile {
+            return false
+        }
+
+        if let uti = try! URL(fileURLWithPath: self.string)
+            .resourceValues(forKeys: [URLResourceKey.typeIdentifierKey])
+            .typeIdentifier {
+            // NOTE: lproj is public.folder
+            return uti != "public.folder"
```

Comment:
If someone looks at this code in three months, it's hard to say what it does. Which types of paths am I ignoring with this expression. Maybe it's worth adding a larger comment where you explain the rationale behind this line.

Reply:
Thanks. Will do. Maybe as a doc-comment.

Reply:
done in 6933cde